src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceColumnManager.java

```diff
@@ -2332,9 +2347,54 @@ public void onError(ServerError error)
       dependencyManager_.withRMarkdown("R Notebook", "Using R Notebooks", extractRmdCommand);
    }
 
+   private void openNotebook(final FileSystemItem rnbFile,
+                             final SourceColumn targetColumn,
+                             final ResultCallback<EditingTarget, ServerError> resultCallback)
+   {
+      // construct path to .Rmd
+      final String rnbPath = rnbFile.getPath();
+      final String rmdPath = FilePathUtils.filePathSansExtension(rnbPath) + ".Rmd";
+      final FileSystemItem rmdFile = FileSystemItem.createFile(rmdPath);
+
+      // if we already have associated .Rmd file open, then just edit it
+      // TODO: should we perform conflict resolution here as well?
```

Comment:
Are you OK leaving this as a TODO, or should it be investigated further now?

Reply:
I believe this TODO already exists in our codebase and just looks new due to git's faulty diffing.

Reply:
Correct, this comment was left in, but because I moved some code around it looks like it was appended.